tests/NuGet.Services.AzureSearch.Tests/SearchDocumentBuilderFacts.cs

```diff
@@ -734,6 +746,102 @@ public async Task SetsExpectedProperties(SearchFilters searchFilters, string exp
 }", json);
             }
 
+            [Theory]
+            [MemberData(nameof(PackageTypesData))]
+            public async Task SetsExpectedPackageTypes(List<PackageType> packageTypes, string expectedFilterable, string expectedDisplay)
+            {
+                var package = Data.PackageEntity;
+                package.PackageTypes = packageTypes;
+
+                var document = _target.FullFromDb(
+                    Data.PackageId,
+                    SearchFilters.Default,
+                    Data.Versions,
+                    isLatestStable: false,
+                    isLatest: true,
+                    fullVersion: Data.FullVersion,
+                    package: package,
+                    owners: Data.Owners,
+                    totalDownloadCount: Data.TotalDownloadCount,
+                    isExcludedByDefault: false);
+
+                SetDocumentLastUpdated(document);
+                var json = await SerializationUtilities.SerializeToJsonAsync(document);
+                Assert.Equal(@"{
```

Comment:
Most of this assertion is covered above. Can we just assert the two properties?

Reply:
Updated to contains assertion for filterablePackageTypes and packageTypes. much more readable.